src/models/Roles/AdminRole.tsx

```diff
@@ -0,0 +1,14 @@
+export enum AdminRole {
+  Admin = 0,          //Admin
+  GoverningBodyHead,  //Голова керівного органу
```

Comment:
please, set all the enum values to some numbers, to make your code more consistent

Reply:
Done. Please, check again